lavalink/__init__.py

```diff
@@ -1,16 +1,6 @@
-import logging
+from .log import set_logging_level, log, socket_log, ws_discord_log, ws_ll_log, ws_rll_log
 
-log = logging.getLogger("red.core.RLL")
-socket_log = logging.getLogger("red.core.RLL.socket")
-socket_log.setLevel(logging.INFO)
-
-ws_discord_log = logging.getLogger("red.Audio.WS.discord")
-ws_ll_log = logging.getLogger("red.Audio.WS.LLServer")
-ws_rll_log = logging.getLogger("red.Audio.WS.RLL")
-log.setLevel(logging.INFO)
-ws_discord_log.setLevel(logging.INFO)
-ws_ll_log.setLevel(logging.INFO)
-ws_rll_log.setLevel(logging.INFO)
+set_logging_level(20)
```

Comment:
The function already has a default value so let's not introduce a magic number here.
```suggestion
set_logging_level()
```